### PR TITLE
docs(project): add project specific docs

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Signed-off-by: Utkarsh Mani Tripathi <utkarsh.tripathi@mayadata.io>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Release Process
+jiva follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/openebs/openebs/projects).
```
Comment (kmova) [inline RELEASE.md:2]: https://github.com/orgs/openebs/projects

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[A clear and concise description of what the bug is, and what commands you ran.)
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
+
+* `kubectl logs <jiva controller pod name> -n openebs`
```
Comment (kmova) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: mayactl volume info command also will be good to fetch here.

Comment (kmova) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: And the status of the pods. 

the users may hesitate to provide the logs.

Comment (utkarshmani1997) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: @kmova ok, i'm putting this as optional.

File: developer-setup.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+```
+
+* To build the docker image
+```
+make build
+```
+
+* Test your changes
+Integration tests are written in shell script and jiva controller and replicas are run as docker containers.
+To run the run the integration tests locally, run
```
Comment (shubham14bajpai) [inline developer-setup.md:55]: nit: `run the` is repeated in the statement.

Comment (utkarshmani1997) [reply developer-setup.md]: addressed

File: developer-setup.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+git rebase upstream/master
+```
+
+While you rebase your changes, you must resolve any conflicts that might arise and build and test your changes using the above steps.
+
+## Submission
+
+### Create a pull request
+
+Before you raise the Pull Requests, ensure you have reviewed the checklist in the [CONTRIBUTING GUIDE](../CONTRIBUTING.md):
```
Comment (kmova) [inline developer-setup.md:168]: can you check this link please

**Pull request merged**